pkg/apis/channels/v1alpha1/subscription_validation.go

```diff
@@ -49,7 +57,7 @@ func (current *Subscription) CheckImmutableFields(og apis.Immutable) *apis.Field
 		return nil
 	}
 
-	ignoreArguments := cmpopts.IgnoreFields(SubscriptionSpec{}, "Subscriber", "Arguments")
+	ignoreArguments := cmpopts.IgnoreFields(SubscriptionSpec{}, "Processor", "Arguments")
```

Comment:
`To` should also allow mutation, only `From` should be immutable. It was an oversight that `ReplyTo` is immutable right now.